@types/ref-array-napi/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for ref-array-napi
```

Comment:
Good idea :) but, I have a doubt when the npm package will be released. Theses files won't be packaged. It means theses types won't be available when a developer install our package.

Reply:
I think to fix it, you have to move this files in src/@types and import it in index.ts? Why do you think ?

Reply:
I'll try it out right now.